themes/my-test-theme-dark/color.yaml

```
@@ -0,0 +1,58 @@
+# This is a sample theme used for demo
+# author: Prune <prune@lecentre.net>
+# last update: 20240324
+
+kubectl: kubectl # path to kubectl executable
+preset: dark # color theme preset
+objFreshThreshold: 1m # ages below this uses theme.data.durationfresh coloring
+
```

Suggest removing configs that use the defaults. To keep the theme minimal.

Also, suggest removing the comments as we don't want to maintain these comments too when we update the docs. Instead only use YAML comments to explain stuff that's specific to this theme


```suggestion
objFreshThreshold: 1m

```

Same goes for the rest of this file. Don't want to spam suggestion comments

---

OK, done.

I'll look into the `go run ./internal/cmd/configdoc` thing... maybe we should have an "empty theme" file with all the current values that someone could use a boilerplate, even if they don't have `go` installed ?

---

I think relying on the config docs in the README.md (in the repo root) should hopefully suffice.